### PR TITLE
Niamh25/doc 5544/move utxo extensions

### PR DESCRIPTION
**Pull request by Niamh25**
Moves the UTXO advanced ledger extensions under the UTXO API folder. Fixed table to add more space to first column.

### CODE REVIEW

**Comment by Niamh25** [comment]
Preview site here: https://niamh.preview.docs.r3.com/en/platform/corda/5.0/developing-applications/api/application/ledger/utxo-ledger/utxo-advanced-ledger-extensions.html

**Review by nadinequinr3 — CHANGES_REQUESTED** [review]
I think the `ledger` folder should just be in the `api` folder (not `api/application`)? It isn't displayed under https://niamh.preview.docs.r3.com/en/platform/corda/5.0/developing-applications/api/application.html

File: content/en/platform/corda/5.0/developing-applications/api/ledger/utxo-ledger/_index.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+title: "net.corda.v5.ledger.utxo"
+menu:
+  corda5:
+    identifier: corda5-api-ledger-utxo
+    parent: corda5-api-ledger
+    weight: 3000
+section_menu: corda5
+---
+
+# net.corda.v5.ledger.utxo
```
Comment (yarivus) [inline content/en/platform/corda/5.0/developing-applications/api/ledger/utxo-ledger/_index.md:13]: shouldn't there be {{childpages}} here ?

**Review by yarivus — COMMENTED** [review]
see 1 comment

**Review by yarivus — COMMENTED** [review]
still pending comments

**Review by nadinequinr3 — APPROVED** [review]

**Review by yarivus — COMMENTED** [review]
thnx @Niamh25

**Pull request merged**